js/scenes/Login/LoginContainer.js

```diff
@@ -61,17 +93,22 @@ LoginContainer.propTypes = {
   dispatch: PropTypes.func.isRequired,
   currentUser: PropTypes.string,
   email: PropTypes.string.isRequired,
-  password: PropTypes.string.isRequired
+  authenticated: PropTypes.bool.isRequired,
+  password: PropTypes.string.isRequired,
+  navigatorUID: PropTypes.string.isRequired,
+  navigator: PropTypes.object.isRequired
 };
 
 LoginContainer.defaultProps = {
```

Comment:
in the js/scenes/Login/Login.js file, you are using currentUser: "" and here you are using currentUser: null. Please be consistent.